Makefile

```diff
@@ -1,6 +1,7 @@
-TEST_INVENTORY ?= tests/inventory.yaml
-TEST_VARS ?= tests/vars.yaml
-TEST_SECRETS ?= tests/secrets.yaml
+TEST_INVENTORY ?= tests/inventory.sample-crc-vagrant.yaml
+TEST_VARS ?= tests/vars.sample.yaml
+TEST_SECRETS ?= tests/secrets.sample.yaml
```

Comment:
I think rather than point to the sample files, we should expect the developer (or CI) to create those files? If there are things that should be reasonably defaulted, then maybe we can move them from the sample files to the defaults.

Reply:
+1 on the addition of TEST_OUTFILE